devlib/module/sched.py

```diff
@@ -195,7 +195,19 @@ class SchedProcFSData(SchedProcFSNode):
 
     @staticmethod
     def available(target):
-        return target.directory_exists(SchedProcFSData.sched_domain_root)
+        path = SchedProcFSData.sched_domain_root
+        cpus = target.list_directory(path) if target.file_exists(path) else []
+
+        if not cpus:
+            return False
+
+        # Even if we have a CPU entry, it can be empty (e.g. hotplugged out)
+        # Make sure some data is there
+        for cpu in cpus:
```

Comment:
I changed that to check at least one CPU has some readable data, I don't know if that's deemed too heavy for probe().

Reply:
This is fine, as this "short circuits" when it finds a populated node. So in the common case, this equivalent to just checking one CPU, but is more robust.